text/0431-guides-restructure.md

```diff
@@ -0,0 +1,414 @@
+- [Dan Monroe](https://github.com/cah-danmonroe)'s trailblazing work to [convert the Guides to use Angle Brackets](https://github.com/ember-learn/guides-source/pull/357)
+- Countless hours of Q&A on Stack Overflow and Discord
+- Experience training new Ember developers in-house
+- Public Ember 101 workshops run in Boston, Massachusetts, USA
+- Studying the guides and tutorials of other JavaScript libraries
+- The successful restructure and rewrite of the [Ember CLI Guides](https://cli.emberjs.com) (a far more drastic project than this RFC aims to be)
+- Attempting to reorder the Guides in a branch, as an experiment
+
+We are confident that this is possible thanks to the incredible response and effort shared by the community for the CLI Guides work.
+
+### Audience
+
+There are many different types of documentation within Ember, so it's important to identify a target audience for each, in order to guide decision making and provide a learning flow that grows with the reader.
+
+There are 3 main audiences for our resources: 
+
+1. newcomers
+2. upgraders
+3. specific-answer-seekers
+
+A huge challenge within the Guides is that they somewhat serve all three, and we'd like to have cleaner distinctions between sections and resources. For example, the Quickstart and Tutorials are distinct resources from the rest of the guides, and they clearly target new users. Upgraders could be served by adding 1 section to the guides as they are today. Specific-answer-seekers must lean on both the API docs and the Guides to find what they need.
+
+We propose that the Guides should focus on the following audience - newcomers who have done the tutorial, and want to build real apps. It will show the "happy path" and how the parts of Ember interact to form good, useful patterns, in a way that would be out of scope for an API documentation block.
+
+Along these lines, anything that can be offloaded to the API docs, should. For example, the Guides should show using a Computed Property/tracked property to solve a problem in a Component or Controller. They should not get into the depths of syntax option for one aspect of an API surface, but rather link to them.
+
+We also have the current hypothesis to inform intro sections, as suggested by [Mike North](https://github.com/mike-north): "People want to get the smallest understandable atom that they can jump in on." This will help us refactor how we present different topics.
+
+### Table of Contents
+
+The following Table of Contents will be applied iteratively over the course of many months. It is not considered a blocker for the release of the Octane edition. The quick wins and urgent Octane refactors will be applied ASAP, and are described in the Implementation section of this RFC. Justification for the ordering is provided following the topics list.
+
+**Core Concepts**
+
+- What is Ember?
+- Getting Started
+- Anatomy of an app (future new content)
```

Comment:
Very valuable point. And just in the right place. 

I would like to see here detailed explanation about ember's folder structure and how this convention enables reusability of addons.

Reply:
Also, since we discuss here folder structure, would be nice to mention here CLI which takes care about boilercode, naming and folders.

Reply:
@chilicoder we could probably add a callout that links to the CLI guide. We purposefully designed this component for just this sort of use case. 👍